api/openapi/v1/support-notifications.yaml

```diff
@@ -57,11 +65,11 @@ paths:
         required: true
       responses:
         202:
-          description: If Alerts and Notifications Service is available, it always
-            returns 202 Accepted to clients with the new ID to indicate the Notification
-            has been received.
+          description: If the Alerts and Notification service is available, it
```

Comment:
Drop all of this preamble. We know it's a 202, we don't need to know what conditions need to exist before the 202 can be emitted.

"Indicates the notification has been received"